infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```
@@ -1,6 +1,25 @@
 metadata name = 'ALZ Bicep - ALZ Default Policy Assignments'
 metadata description = 'This module will assign the ALZ Default Policy Assignments to the ALZ Management Group hierarchy'
 
+type policyAssignmentSovereigntyGlobalOptionsType = ({
+  @sys.description('Switch to enable/disable deployment of the Sovereignty Baseline - Global Policies Assignment to the intermediate root management group.')
+  parTopLevelSovereigntyGlobalPoliciesEnable: bool
+
+  @sys.description('The list of locations that your organization can use to restrict deploying resources to. If left empty, only the deployment location will be allowed.')
+  parListOfAllowedLocations: string[]
+})
```

```suggestion
type policyAssignmentSovereigntyGlobalOptionsType = {
  @sys.description('Switch to enable/disable deployment of the Sovereignty Baseline - Global Policies Assignment to the intermediate root management group.')
  parTopLevelSovereigntyGlobalPoliciesEnable: bool

  @sys.description('The list of locations that your organization can use to restrict deploying resources to. If left empty, only the deployment location will be allowed.')
  parListOfAllowedLocations: string[]
}
```
dont think the parentheses are needed